examples/k8petstore/k8petstore.sh

```diff
@@ -16,16 +16,32 @@
 
 echo "WRITING KUBE FILES , will overwrite the jsons, then testing pods. is kube clean ready to go?"
 
+
```

Comment:
Missing a control header to keep the script from running away from the user. Typically this is done with a header like set -eu at the top of the file. Where there are cases you want errors to go without bailing on the user, set +e for the context of that command.